packages/orm/src/index.ts

```diff
@@ -319,7 +319,7 @@ export class ConfigApi<T extends Entity, C> {
     }
   }
 
-  beforeDelete<H extends LoadHint<T>>(populate: H, fn: HookFn<Loaded<T, H>, C>): void;
+  beforeDelete<H extends LoadHint<T>>(popuate: H, fn: HookFn<Loaded<T, H>, C>): void;
```

Comment:
not sure why this changed? `populate` is now mispelled

Reply:
Yep, fixed in #213